cli/test/test_common.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+import shutil
+import contextlib
+import subprocess
+import tempfile
+import git
+import unittest
+
+FIXDIR = f'{os.path.dirname(os.path.realpath(__file__))}/fixtures'
+
+
+class PopperTest:
```

Comment:
one last change it just ocurred to me. We can derive from `unittest.TestCase` here so that children classes only derive from `PopperTest`

Reply:
Updated for all those classes that were inheriting the PopperTest class.